src/main/g8/$if(mill.truthy)$build.mill$endif$

```diff
@@ -0,0 +1,57 @@
+package build
+
+import mill._, scalalib._, publish._
+
+object Deps {
+  val ScalaVersion = "2.13.15"
+
+  val Http4sVersion = "0.23.30"
+  val CirceVersion = "0.14.10"
+  val MunitVersion = "1.1.0"
+  val LogbackVersion = "1.5.16"
+  val MunitCatsEffectVersion = "2.0.0"
+
+  def http4s(module: String): Dep = ivy"org.http4s::http4s-\$module:\$Http4sVersion"
+}
+
+object `package` extends RootModule with SbtModule with PublishModule {
+  import Deps._
+
+  def scalaVersion = ScalaVersion
+
+  def artifactName = "$name;format="norm"$"
+
+  def publishVersion = "0.0.1-SNAPSHOT"
+
+  def pomSettings = PomSettings(
+    description = artifactName(),
+    organization = "$organization$",
+    url = "https://http4s.org/v0.23/docs/quickstart.html#giter8-template",
```

Comment:
This should be dropped, that's the info from the outer build.sbt and not what we want to generate in the template

Reply:
All PomSettings fields are required. The idea was to add support for publishing this dependency as a jar, when removed there is no more publish tasks available (`mill publishLocal`) and the default output from `mill jar` is `out/jar.dest/out.jar`. I am removing it.